### PR TITLE
Adds ASMAT2 info

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -317,4 +317,5 @@ AJ20230507,Adolf Jensen,2023-05-07T00:00:00,2023-05-11T23:59:59,Irminger glider
 MC20230508,Monte Carlo,2023-05-08T14:32:00,2023-05-08T20:58:00,Endurance GHS glider deployment 382-13
 EK20230510,Elakha,2023-05-10T16:00:00,2023-05-10T21:22:00,Endurance ce01issp-00021 CSPP deployment and glider recovery.
 SKQ202308S,Sikuliaq,2023-05-13T16:46:00,2023-05-27T15:48:00,Papa 10
+KH20230608,Kahuna,2023-06-08T11:00:00,2023-06-08T23:00:00,ASMAT2 test glider deployments
```
Comment (snwhite) [inline cruise/CruiseInformation.csv:320]: Looks good.

File: vessel/shiplist.csv [hunk 2]
```diff
@@ -64,6 +65,6 @@ WT,R/V,Thomas Washington,31WT,SIO,KGWU,,,63.7,13.8,4.9,AGOR-10,(N),Y
 TA,R/V,Tioga,,WHOI,,,,,,,,Y,N
 V,R/V,Vema,31VE,LDEO,J8RU,,7738383,,,,,(N),Y
 WJ,M/V,Warren Jr.,,BHC,,,,,,,,Y,N
-WOO,F/V,Wahoo,,Meredith,,,,,,,,Y,N
+WOO,F/V,Wahoo,,,,,,,,,,Y,N
 W,R/V,Wecoma,32WC,OSU,WSD7079,303014000,7604439,56.4,7.2,6,,(N),Y
 ZP,R/V,Zephyr,,Zephyr Northwest,WDG9976,338142000,8992845,24,5.1,2.4,,Y,N
```
Comment (snwhite) [inline vessel/shiplist.csv:70]: Both look good.

**Review by snwhite — APPROVED** [review]

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (cooleyky) [inline cruise/CruiseInformation.csv:320]: Looks good

File: vessel/shiplist.csv [hunk 3]
```diff
@@ -64,6 +65,6 @@ WT,R/V,Thomas Washington,31WT,SIO,KGWU,,,63.7,13.8,4.9,AGOR-10,(N),Y
 TA,R/V,Tioga,,WHOI,,,,,,,,Y,N
 V,R/V,Vema,31VE,LDEO,J8RU,,7738383,,,,,(N),Y
 WJ,M/V,Warren Jr.,,BHC,,,,,,,,Y,N
-WOO,F/V,Wahoo,,Meredith,,,,,,,,Y,N
+WOO,F/V,Wahoo,,,,,,,,,,Y,N
```
Comment (cooleyky) [inline vessel/shiplist.csv:68]: Looks good

**Review by cooleyky — APPROVED** [review]

**Pull request merged**